composer.json

```diff
@@ -31,13 +31,12 @@
         "mockery/mockery": "^1.4",
         "pamil/prophecy-common": "^0.1",
         "phpspec/phpspec": "^6.1",
-        "phpstan/phpstan": "0.12.29",
-        "phpstan/phpstan-doctrine": "0.12.16",
+        "phpstan/phpstan": "^0.12.57",
+        "phpstan/phpstan-doctrine": "^0.12.19",
```

Comment:
Are you sure that we break nothing with that ?

Reply:
it's the minimal versions that allow php8
Sylius is even highter: 
https://github.com/Sylius/Sylius/blob/c2d97c8dd44530d59d588f41498042950f801c20/composer.json#L188